src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -24,11 +20,10 @@ export interface SchemaRowProps {
 }
 
 export const SchemaRow: React.FunctionComponent<SchemaRowProps> = ({ schemaNode, nestingLevel }) => {
-  const { defaultExpandedDepth, renderRowAddon, onGoToRef, hideExamples } = useJSVOptionsContext();
+  const { defaultExpandedDepth, renderRowAddon, onGoToRef, hideExamples, renderRootTreeLines } = useJSVOptionsContext();
 
-  const [isExpanded, setExpanded] = React.useState<boolean>(
-    !isMirroredNode(schemaNode) && nestingLevel <= defaultExpandedDepth,
-  );
+  const setPathCrumbs = useUpdateAtom(pathCrumbsAtom);
+  const [isExpanded, setExpanded] = React.useState<boolean>(nestingLevel <= defaultExpandedDepth);
```

Comment:
`isMirroredNode(schemaNode)` bit is super important, we need to add it back
```suggestion
  const [isExpanded, setExpanded] = React.useState<boolean>(!isMirroredNode(schemaNode) && nestingLevel <= defaultExpandedDepth);
```

We'll enter an infinite loop if `defaultExpandedDepth` is set to `Infinity` which actually might be the case somewhere in platform 😬.

If you do want them to be expanded, then perhaps let's add such a check

```suggestion
  const [isExpanded, setExpanded] = React.useState<boolean>((Number.isFinite(defaultExpandedDepth) || !isMirroredNode(schemaNode)) && nestingLevel <= defaultExpandedDepth);
```

Reply:
Let me remind myself why I did this change, one sec :).

Reply:
You're right, nice catch - fixed.